drivers/adc/ad7779/ad7779.h

```diff
@@ -271,12 +272,13 @@ typedef struct {
 	ad7779_state		sar_state;
 	ad7779_sar_mux		sar_mux;
 	ad7779_state		sinc5_state;	// Can be enabled only for AD7771
+	bool				read_from_cache;
 	uint8_t			cached_reg_val[AD7779_REG_SRC_UPDATE + 1];
 } ad7779_dev;
 
 typedef struct {
 	/* SPI */
-	struct no_os_spi_init_param		spi_init;
+	no_os_spi_init_param		spi_init;
```

Comment:
do not remove the explicit declaration (`struct`), we removed the typedefs with https://github.com/analogdevicesinc/no-OS/commit/b21f3caad8f1ce1296e51ade11259b082bf3c361

Reply:
Reverted this- struct has been added back now